src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -75,7 +74,7 @@ object Run  {
 
       // Update the database state:
       val newBackendInfo = JesCallBackendInfo(db.CallStatus(ExecutionStatus.Running.toString), Option(JesId(run.jesId)), Option(JesStatus(currentStatus.toString)))
-      dataAccess.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
+      DataAccess.instance.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
```

Comment:
Oh I didn't want to extend the trait, I was looking for a way to accomplish this with implicit magic.  And basically I failed, so never mind.  :frowning: